rosplane/scripts/rosplane_gcs_launch.sh

```diff
@@ -0,0 +1,196 @@
+    echo "Usage: $0 [-h] [-s] [-t] [-r] [-a aircraft] [-b [bag_name]] [-o user@host] [-d docker_container_name] path/to/ROSflight/workspace"
+}
+
+sim=false
+tuning=false
+rc_sim=false
+aircraft='anaconda'
+online=false
+docker=false
+bag=false
+
+# Parse options using getopts
+while getopts ":hstra:b:o:d:" opt; do
+    case $opt in
+        h)
+            
+            print_help
+            exit 0
+            ;;
+        s)
+            sim=true
+            ;;
+        t)
+            tuning=true
+            ;;
+        r)
+            rc_sim=true
+            if ! $sim; then
+              echo "Cannot run simulated transitter, while not running a simulation."
+              exit 1 
+            fi
+            ;;
+        a)
+            aircraft=$OPTARG
+            ;;
+        b)
+            bag=true
+            bag_name=$OPTARG
+            ;;
+        o)
+            online=true
+            user_host=$OPTARG
+            if $sim; then
+              echo "Cannot run simulat on a remote."
+              exit 1 
+            fi
+            ;;
+        d)
+            docker=true
+            container=$OPTARG
+            if ! $online; then
+              echo "Script not configured for Docker not on remote."
+            fi
+            ;;
+        \?)
+            echo "Invalid option: -$OPTARG" >&2
+            print_help
+            exit 1
+            ;;
+    esac
+done
+
+shift $((OPTIND -1))
+
+# Get the file path
+filepath=$1
+
+# Check if filepath is provided
+if [ -z "$filepath" ]; then
+    echo "Absolute/relative file path is required"
+    print_help
+fi
+
+# Create a new tmux session
+tmux new-session -d -s rosplane_sim_session
+
+tmux source-file ~/.tmux.conf
+
+# Split the tmux window into 4 panes
+tmux split-window -t rosplane_sim_session:0.0 -h
+tmux split-window -t rosplane_sim_session:0.0 -v
+tmux split-window -t rosplane_sim_session:0.2 -v
+
+# Arrange panes in each corner
+tmux select-pane -t rosplane_sim_session:0.0
+tmux select-pane -t rosplane_sim_session:0.1
+tmux select-pane -t rosplane_sim_session:0.2
+tmux select-pane -t rosplane_sim_session:0.3
+
+# Window placement reference:
+# rosplane_sim_session:0.0 Top Left
+# rosplane_sim_session:0.1 Bottom Left
+# rosplane_sim_session:0.2 Top Right
+# rosplane_sim_session:0.3 Bottom Right
+
+# Setup all panes to ssh into the remote
+if $online; then
+  tmux send-keys -t rosplane_sim_session:0.0 "ssh $user_host" C-m
+  tmux send-keys -t rosplane_sim_session:0.1 "ssh $user_host" C-m
+  tmux send-keys -t rosplane_sim_session:0.2 "ssh $user_host" C-m
+  tmux send-keys -t rosplane_sim_session:0.3 "ssh $user_host" C-m
+fi 
+
+# Send all of the panes to the working directory.
+tmux send-keys -t rosplane_sim_session:0.0 "cd $filepath" C-m
+tmux send-keys -t rosplane_sim_session:0.1 "cd $filepath" C-m
+tmux send-keys -t rosplane_sim_session:0.2 "cd $filepath" C-m
+tmux send-keys -t rosplane_sim_session:0.3 "cd $filepath" C-m
+
+# TODO test docker implementation.
```

Comment:
We should test this or remove the comment before merging.

Reply:
Oh haha I'll remove it I tested it.